src/ape/managers/base.py

```diff
@@ -0,0 +1,5 @@
+from ape.utils import ManagerAccessBase
+
+
+class ManagerBase(ManagerAccessBase):
+    pass
```

Comment:
Note: this is to allow us to add other IPython integration features for managers